packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -224,6 +224,7 @@ class WebView extends StatefulWidget {
     this.onPageStarted,
     this.onPageFinished,
     this.onProgress,
+    this.onPullToRefresh, //@pullToRefresh
```

Comment:
What are these inline comment annotations throughout the PR? This is not something we would land.